cli/embedded-files/tekton/ingress.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: traefik
+  name: tekton-dashboard
+  namespace: tekton-pipelines
+spec:
+  rules:
+    - host: "tekton.{{PUBLIC_IP}}.nip.io"
```

Comment:
Missed this. the Tekton deployment needs a bit more code to handle this: https://github.com/SUSE/carrier/blob/main/tekton/install#L61-L63

Reply:
Added here: https://github.com/SUSE/carrier/pull/35/commits/22bf48d95a02d62d2ef988e4cd2918f298c3b7a0